packages/grafana-e2e-selectors/src/types/selectors.ts

```diff
@@ -1,42 +1,67 @@
 /**
  * A string selector
- *
- * @alpha
  */
 
 export type StringSelector = string;
 
 /**
- * A function selector with an argument
- *
- * @alpha
+ * A function selector with one argument
  */
 export type FunctionSelector = (id: string) => string;
 
 /**
- * A function selector without argument
- *
- * @alpha
+ * A function selector with two arguments
  */
-export type CssSelector = () => string;
+export type FunctionSelector2 = (arg1: string, arg2: string) => string;
```

Comment:
nit: can we name this something better. even just `FunctionSelector2Args` or `FunctionSelectorAlt` or something 😅